data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -171,6 +171,7 @@ A poisoned unit cannot be cured of its poison by a healer, and must seek the car
         female_name= _ "female^formation"
         description= _ "This unit gains a +10% bonus to defense when another unit with the same ability is adjacent to it. However, this cannot raise the unit’s defense above 70%."
         special_note=_"Groups of units of this type are able to shield each other in combat."
+        describe_effect=_"cth decreased"
```

Comment:
I'm guessing 'cth' is 'Chance to Hit'. Why not write it out completely?

Reply:
Like it must replace 'name' in attack prediction windows, describe_effect' must be have two word maximum else ithis would be too heavy, this is why i use abreviation, if you're suggestion, don't hesitate.

Reply:
I understand English isn't your native language, but I'm not sure I understood all of that. Two word limit sounds very tight... and arbitrary. But I suppose if it really needs to be abbreviated, then capitalise CTH to make it clear (at least in English convention) that it's an acronym.

Reply:
Using chance to hit in any letter would be too cumbersome in the attack prediction window but yes I should use CTH in all capitals then, unless you have a better description to come up with (one or two words to avoid line breaks)